internal/cli/cmd/verify/stage.go

```diff
@@ -0,0 +1,134 @@
+package verify
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"strings"
+
+	"connectrpc.com/connect"
+	"github.com/spf13/cobra"
+
+	"github.com/akuity/kargo/internal/cli/client"
+	"github.com/akuity/kargo/internal/cli/config"
+	"github.com/akuity/kargo/internal/cli/option"
+	v1alpha1 "github.com/akuity/kargo/pkg/api/service/v1alpha1"
+)
+
+type verifyStageOptions struct {
+	Config        config.CLIConfig
+	ClientOptions client.Options
+
+	Project string
+	Name    string
+	Abort   bool
+}
+
+func newVerifyStageCommand(cfg config.CLIConfig) *cobra.Command {
+	cmdOpts := &verifyStageOptions{
+		Config: cfg,
+	}
+
+	cmd := &cobra.Command{
+		Use:   "stage [--project=project] (NAME) [--abort]",
+		Short: "(Re)run or abort the verification of the stage's current freight",
+		Args:  option.ExactArgs(1),
+		Example: `
+# Rerun the verification of the stage's current freight
+kargo verify stage --project=my-project my-stage
+
+# Rerun the verification of a stage in the default project
+kargo config set-project my-project
+kargo verify stage my-stage
+
+# Abort the verification of a stage's current freight
+kargo verify stage --project=my-project my-stage --abort
+
+# Abort the verification of a stage in the default project
+kargo config set-project my-project
+kargo verify stage my-stage --abort
+`,
```

Comment:
Nit: We need to fix this in a bunch of places. We're getting an extra line break at the end of the examples section (two instead of one), so we should probably move these ending ticks to the end of the previous line.

Not a blocker. Just a thing I've noticed -- and definitely not unique to this PR.

Reply:
It also actually has a leading newline everywhere. We should probably just do:

```
Example: `# Verify a stage
kargo verify stage --project=my-project my-stage`,
```

Reply:
> It also actually has a leading newline everywhere.

Funny... I noticed the double blank line after examples and never noticed that the _one_ blank line between "Examples:" and the actual examples was still one blank line more than what is between all the other help section headings and their content.

Good eye.

Reply:
What I have see some projects do is adding a tiny wrapper which adds `strings.TrimSpace` (e.g. `printDescription`), as this allows you to continue to maintain the visible advantage of having everything start at the same point in-code and/or allow you to move it to constants defined outside of the struct.